gatsby/content/blog/2023/02/2023-02-28-synapse-1.78.mdx

```diff
@@ -0,0 +1,53 @@
+---
+date: '2023-02-28'
+title: Synapse 1.78 released
+categories:
+- Releases
+author: H. Shay
+---
+
+Greetings Matrix fans! We've published Synapse [version 1.78](https://github.com/matrix-org/synapse/releases/tag/v1.78.0)
+as the new stable release this week. Synapse admins are encouraged to upgrade
+to it at their convenience. Please take a look at the [upgrade notes](https://matrix-org.github.io/synapse/v1.78/upgrade.html#upgrading-to-v1780)
+for any important information about upgrading.
+
+## Announcements
+
+Please note that Synapse 1.78.0 replaces the `/_synapse/admin/v1/media/<server_name>/delete` admin API with an identical
+endpoint at `/_synapse/admin/v1/media/delete`. Please update your tooling to use the new endpoint. The deprecated version
+will be removed in a future release.
+
+## The new stuff
+
+In case you were unaware, Synapse has a command line export data command which allows administrators to export data for
+a specific user (you can read more about this command [here](https://matrix-org.github.io/synapse/latest/usage/administration/admin_faq.html#how-can-i-export-user-data)).
+Synapse 1.78.0 updates this command adding account data to the user information that is returned by the export data command.
+
+This version of Synapse also features a few changes to push rules, with implementations for [MSC3758](https://github.com/matrix-org/matrix-spec-proposals/pull/3758),
+[MSC3966](https://github.com/matrix-org/matrix-spec-proposals/pull/3966), and the removal of the spurious `dont_notify`
+action from the defaults for the `.m.rule.reaction` pushrule.
```

Comment:
can we possibly expand on what this means for users? I don't understand what it means to be honest and I thought I knew about push rules!

Reply:
lol honestly I couldn't really entirely parse out what it meant for end users either but I needed to write about something...

Reply:
Basically they are fundamental tools that give people greater flexibility in customising their notification rules. With MSC3758, you can now ask to be notified when a field in an event contains any given type of value. Before that, you could only match on strings.

And MSC3966 allows for sending a notification when an item contained in an array value in an event matches a given pattern. This is a change that's necessary for MSC3952 - specifically to see if your Matrix ID has been included in a new `m.mentions.user_ids` field on an event, which that MSC defines.

Reply:
Thanks for the insight anoa - I am going to nick some of this explanation for the blog :)